packages/backend/src/graphql/schema.graphql

```diff
@@ -1025,6 +1045,14 @@ input CreateWalletAddressInput {
   publicName: String
   "Unique key to ensure duplicate or retried requests are processed only once. See [idempotence](https://en.wikipedia.org/wiki/Idempotence)"
   idempotencyKey: String
+  "Additional properties associated with the [walletAddress]."
+  additionalProperties: [AdditionalPropertyInput!]!
```

Comment:
I think the `additionalProperties` field on the wallet address create input should be optional. just makes it easier on the caller. So this would be:

```gql
additionalProperties: [AdditionalPropertyInput!]
```

then we can remove the empty `additionalProperties` arrays where the mutation is called. like `test/integration/lib/integration-server.ts`, `packages/mock-account-service-lib/src/requesters.ts`, `packages/frontend/app/routes/wallet-addresses.create.tsx`